htdocs/web_portal/views/admin/add_service_type.php

```diff
@@ -6,8 +6,14 @@
         <input type="text" value="" name="Name" class="input_input_text">
         <span class="input_name">Description</span>
         <input type="text" value="" name="Description" class="input_input_text">
-
+        <span class="input_name" style="">
+            <input type="checkbox" name="AllowMonitoringException" value="checked"/>
```

Comment:
Can you add ID(Unique identifier) for INPUT  element?  So that label can be bound to the input

Reply:
What is the "label" in this case?

Reply:
`ID` helps us to identify the element uniquely in the DOM. So, When we are using `label` with `for`. It helps us make a connection between those. For instance, Say another developer declared with the same name as name="AllowMonitoringException. Now, We have two input fields with name="AllowMonitoringException" our `label` field will be confused to make a connection between them. Hope it make sense?

Reply:
Found this helpful: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Reply:
ah okay, I wasn't seeing the line below this one where `label for` was being used.

What happens if we have the same name but different IDs? Or is that against convention/good html practices - i..e `name`, `label` and `id` should be the same?

Reply:
IMO: yes, Greg. Because it helps the browser when JS code is loaded as well as the users, usually, the ID and label name should be the same. And the name for the input field helps us to fetch and play with the value(Under the hood: `Name` implies `form control`). 
To answer whether name, label and id should be the same?
ID and label name should be the same always. The name helps to watch or change the form control name(Input field in our case). Ideally, we should NOT use the same input name because it might create an issue when it is performing a lookup for the browser. (We can handle that case with form arrays if the input name is the same). Does it make sense again?

Reply:
This comment is just meant to indicate that I have updated my comments for better readability.

Reply:
thanks for the clarification on this 😄, I think we should address it across the codebase as a whole. @Sae126V can you open a seperate issue to track this?

Reply:
Cool. Code is looking good to me. So, I am resolving the issues then.

Reply:
Tracked in #450